app/src/main/java/com/example/moviedb/ui/popular/MovieAdapter.kt

```diff
@@ -6,18 +6,28 @@ import com.example.moviedb.data.model.Movie
 import com.example.moviedb.databinding.ItemMovieBinding
 import com.example.moviedb.ui.base.BaseAdapter
 
-class MovieAdapter : BaseAdapter<Movie, ItemMovieBinding>(object : DiffUtil.ItemCallback<Movie>() {
+class MovieAdapter(val itemClick: (Movie) -> Unit = {}) :
+    BaseAdapter<Movie, ItemMovieBinding>(object : DiffUtil.ItemCallback<Movie>() {
 
-    override fun areItemsTheSame(oldItem: Movie, newItem: Movie): Boolean {
-        return oldItem.id == newItem.id
-    }
+        override fun areItemsTheSame(oldItem: Movie, newItem: Movie): Boolean {
+            return oldItem.id == newItem.id
+        }
 
-    override fun areContentsTheSame(oldItem: Movie, newItem: Movie): Boolean {
-        return oldItem.id == newItem.id && oldItem.title == newItem.title
-    }
-}) {
+        override fun areContentsTheSame(oldItem: Movie, newItem: Movie): Boolean {
+            return oldItem.id == newItem.id && oldItem.title == newItem.title
```

Comment:
content the same thì nên so sáng những cái hiển thị lên ui của item_movie

Reply:
Dạ vâng ạ